### DISCUSSION TITLE
Visibility Setting/Binding Issue in  11.2

### DISCUSSION THREAD

**Discussion by pawfectcat** [AvaloniaUI/Avalonia#17429] [General]
After upgrading to 11.2, all components/buttons are displayed simultaneously irrespective of state/visibility setting.

It seems to be either a binding issue or a visibility setting issue.

Does anyone know how to fix this? Thank you

**Comment by stevemonaco** [comment]
Haven't heard of this issue. Ensure you've updated all Avalonia core packages to 11.2. If that doesn't work, please create a minimal sample that reproduces the problem.

**Reply by StefanKoell** [reply]
I've also seen some changes in behavior. I used to have a IsVisible binding to a boolean using #ElementName.Class.Property which worked fine with 11.2.0-rc2 but did not work anymore after upgrading to 11.2.0. Had to change it to $parent[Type].DataContext.Property (with casting).

@pawfectcat maybe you can provide more information on how you set up the binding in your code.
